docs/guide/Configs/Exporters.md

```diff
@@ -207,8 +207,7 @@ Example of `IntroJsonExport-report-brief.json`:
 
 ## Plots
 
-If you have installed [R](https://www.r-project.org/), defined `%R_HOME%` variable and used `RPlotExporter.Default` and `CsvMeasurementsExporter.Default` 
-in your config, you will also get nice plots with help of the `BuildPlots.R` script in your bin directory. 
+If you have installed [R](https://www.r-project.org/), defined a `%R_HOME%` variable pointing to the directory containing `Rscript.exe`, and used `RPlotExporter.Default` and `CsvMeasurementsExporter.Default` in your config, you will also get nice plots with help of the `BuildPlots.R` script in your bin directory.
```

Comment:
We shouldn't cover all the distributions, but it will be nice to have several examples.